### DISCUSSION TITLE
Achievments not Showing

### DISCUSSION THREAD

**Discussion by HamzaMasoodh** [community/community#67590] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi
I have been on github for alomost 2 years
I checked settings my achievements is enabled but on my profile i m unable to see any achievments
I have merged 16 pull requests 
Usually you get 2 achiecements of Pull Shakr at this point But i m unable to see any

**Comment by seyLu** [comment, marked as answer]
#### Hi @HamzaMasoodh,

To get the `Pull Shark` achievement, you must be the author of the pull request and have it merged to the default branch (usually `main`). Merged PRs that you did not open do not count. Closed PRs that are not merged do not count.

You can refer to this [repo](https://github.com/drknzz/GitHub-Achievements) for an overview on github achievements. 

To show achievements, go to settings > public profile and tick the checkbox:
![image](https://github.com/community/community/assets/98249191/1b6d2416-1d93-4514-a8ba-645e05bfb4b0)

Then individually, you can choose to show or hide each achievement on profile page > achievements:
<table>
<td>
<img src="https://github.com/community/community/assets/98249191/972d19fe-1a6f-4991-a8a2-f5627c9d20c3">
</td>
<td>
<img src="https://github.com/community/community/assets/98249191/23d3ef16-f5ab-4809-9259-5f865ecc75cc">
</td>
</table>

**Reply by HamzaMasoodh** [reply]
Hi @seyLu 
My Achievements in the profile tab are already active
<img width="810" alt="Screenshot 2023-09-21 at 7 15 53 AM" src="https://github.com/community/community/assets/65071311/860c1d66-3458-4f21-a3c3-2178ea905508">

For Pull Shark, I have merged the pull requests authored by me.
I didn't even get any other achievements like Quickdraw or YOLO

**Comment by radar-blip** [comment]
They may be hidden for some reason.
Check here to see if you have any achievement badges: https://github.com/HamzaMasoodh?tab=achievements

**Comment by HamzaMasoodh** [comment]
@linuxlawson 
I navigated to this page and got a 404 Not Found error
Can you explain any reason this might be hidden for me
Can I send this issue to Git Hub by any means so this can be resolved?

**Reply by radar-blip** [reply]
That is the tab where all of your achievements would be shown.
Sometimes achievement badges seem to get hidden by default.
You could try one of the categories [here](https://support.github.com/request) for help, or use the search feature.

**Reply by HamzaMasoodh** [reply]
@linuxlawson 
![Github](https://github.com/community/community/assets/65071311/d1ab8019-3372-4815-8869-c13db9fd1323)
This is my Achivement page.
Is there any way to troubleshoot this?

**Reply by radar-blip** [reply]
Your achievements are now showing on your profile.
The achievements link should be working for you also.

https://github.com/HamzaMasoodh?tab=achievements

![Screenshot from 2023-09-27 03-43-14](https://github.com/community/community/assets/39196767/fe88a0d4-6a45-4bd3-a64e-062117880e58)

**Reply by oliverjhn** [reply]
@linuxlawson I also have this issue, my tab returns an error 404, and I'm confident that I've at least got one achievement. Even if I didn't I don't think the page should be a 404. Can you fix it please? Any help would be much appreciated.

![image](https://github.com/community/community/assets/107288810/6853b37e-c1e4-46e1-839c-acac64ead1e4)

**Reply by EHEGUY** [reply]
any updates ??? on this facing the same error

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @HamzaMasoodh. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by hariharasudhan-29507** [comment]
Hi! I am facing the same issue , if i disables show achievements option it shows ,you have unlocked achievements , enable to display them , if i enable that , i am seeing achivement column in my profile 
<img width="1828" height="526" alt="Screenshot 2026-03-27 171219" src="https://github.com/user-attachments/assets/a89f95a9-4937-46b9-aa5e-75e45de6565c" />
even
<img width="1919" height="1038" alt="Screenshot 2026-03-27 173008" src="https://github.com/user-attachments/assets/6f84508a-a4ee-431c-8c28-8ed518a5827a" />
 
<img width="1919" height="888" alt="Screenshot 2026-03-27 173535" src="https://github.com/user-attachments/assets/c3de36b7-15bc-4e93-9235-e57941a09933" />


 i am confident i saw my achievement today morning but they are missing right now

**Reply by EHEGUY** [reply]
any solution 
???

**Comment by AmoolyaSuneja** [comment]
try these : 
1)clear cookies for github.com
2)try a different browser or even a different device
3)try hitting the api directly and check for any possible errors
4)DNS/CDN might be stale. Try using 1.1.1.1 or 8.8.8.8

let me know if anything works!
